requirements/runtime.txt

```diff
@@ -11,4 +11,7 @@ pytorchcv==0.0.55
 tqdm
 onnx==1.6.0
 onnxruntime==1.1.2
-lxml
+lxml==4.5.0
+pycocotools==2.0.0
+tensorboardX==1.8
```

Comment:
We have already have `tensorboard` in optional dependencies. Do you really need `tensorboardX` here?

Reply:
Ok, moved to optional

Reply:
The question was, if you really need `tensorboardX` at all? PyTorch should have a native tensorboard support. See [here](https://github.com/open-mmlab/mmcv/blob/b7e8d7d7fb49ae7a66fce7e6ee1f0cbaf561e7d6/mmcv/runner/hooks/logger/tensorboard.py#L25-L41) for example.

Reply:
When i have tested it on [face-detection](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/object_detection/face_detection.md#4-training-and-fine-tuning) with '--validate', it crashed with `ModuleNotFoundError: No module named TensorboardX`. Now works...